test/stream-serializers.test.js

```diff
@@ -1,18 +1,17 @@
 'use strict'
 
-const t = require('tap')
-const test = t.test
+const { test } = require('node:test')
 const Fastify = require('..')
 const Reply = require('../lib/reply')
 
-test('should serialize reply when response stream is ended', t => {
+test('should serialize reply when response stream is ended', (t, done) => {
   t.plan(3)
   const stream = require('node:stream')
   const fastify = Fastify({
     logger: {
       serializers: {
         res (reply) {
-          t.type(reply, Reply)
+          t.assert.strictEqual(reply instanceof Reply, true)
```

Comment:
Can you add this below this assertion based on https://github.com/fastify/fastify/pull/5839#issuecomment-2472725756. Also udpate t.plan(4) to validate `t.assert.ok()` is tested
```
t.assert.ok('passed')
```

Reply:
if I put `t.plan(4)` gives me `plan expected 4 assertions but received 5`, however if I put `t.plan(5)` the test passes

Reply:
whoops thats my mistake. Sorry `t.plan(5)` is correct since we are streaming reponse.
These two lines should produce the additional 2 asserts on https://github.com/fastify/fastify/blob/main/test/stream-serializers.test.js#L26-L27